react-validated-input-component/my-app/src/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { FaTimes } from 'react-icons/fa';
+import { ImCheckmark } from 'react-icons/im';
+import './Form.css';
+
+export default function Form() {
+  const [password, setPassword] = useState('');
+
+  return (
+    <>
+      <label>
+        Password
+        <div>
+          <input
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+            type="password"
+            name="password"
+            className="input"
+          />
+        </div>
+      </label>
+      <span>
+        {password === '' ? <FaTimes className="wrong" /> : null}
```

Comment:
This all works. Nice!
It would be good to get all the logic on whether or not the password is valid into one place. As you have it here, you have 3 or 4 different places where you have the valid logic repeated. Can you figure out how to put it in one place?